### ISSUE TITLE
Asikart QuickIcons Pro

### ISSUE THREAD

**Issue by wbuk** [asikart/quickicons#41]
Two issues found with Asikart Quick Icons Pro (but I love this extension)

Issue 1
When you click on Images Manger you get a error that says:-

Invalid jQuery UI Images configuration. Selectable, draggable and droppable components must be included.

Issue 2
Also, Three are icons in the CORE category menu that I cant seem to move to others because they are not on the icon list,  They appear to be icons that were created automatically when when I installed QuickIcons Pro.  It has probably detected that Hika Shop, jComments, Checking Joomla, User Points, JCE File Browser, checking extensions, and Akeeba Backup.... this means I cant hide them when some people log in and I also cant move them into a category.

**Comment by asika32764** [comment]
Thank you for reporting this issue, I will try to fix it in next version

**Comment by wbuk** [comment]
This is an automated reply to your message "Re: [lyrasoft/quickicons] Asikart QuickIcons Pro (#41)" sent to sales@webbuster.co.uk.

Dear Simon Asika <notifications@github.com>

Thank you for contacting Web Buster UK Ltd, we will be in touch with you shortly.

**Comment by wbuk** [comment]
Hi

Did you manage to look at this issued, when will the next update be
available?  Also:-

NEW FEATURES REQUEST
Feature 1:-
Any chance you an add Alt tags to the buttons so that when you hover
over them you can assign a longer custom message to that which is on the
buttons.  So this message would need to wrap the text so you can have
several lines.

Feature 2:-
Add Joomla Updater so we are informed of all updates via joomla.

The previous issues mentioned were:-
Problem 1:-
Some buttons like Hika Shop etc appear automatically and you cant remove
them.  Adding automatically is brillaint, I love it but we need to be
able prevent some customers from seeing the buttons, we need to beable
to change the button if we want or move them to another tab just like we
can with all the other buttons we create.

Problem 2:-
When you click on Image Manager on the side menu you get the following
error.
Invalid jQuery UI configuration. Selectable, draggable and droppable
components must be included.

I know these are mostly bug fixes but I would be so greatful if you
could fix them in the PRO version, which is the one we use.

Thank you
On 23/08/2016 00:59, Simon Asika wrote:

> Thank you for reporting this issue, I will try to fix it in next version 
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or mute the thread [2].

-- 
Al Metcalfe
Web Buster UK Ltd
www.webbuster.co.uk
sales@webbuster.co.uk

 

Links:
------
[1]
https://github.com/lyrasoft/quickicons/issues/41#issuecomment-241587486
[2]

**Comment by wbuk** [comment]
Hi

sorry, I found anther bug, when I go to edit a button I get a yellow
message:-

WARNING

JUser: :_load: Unable to load user with ID: 908

Any idea? 
-- 
Al Metcalfe
Web Buster UK Ltd
www.webbuster.co.uk
sales@webbuster.co.uk

**Comment by kingfisher64** [comment]
I'm getting the same errors as reported by wbuk plus one additional

Clicking System > Control Panel > the page formatting is broken. Disable module it displays properly again, enable and breaks again.

When you click on Image Manager on the side menu you get the following
error. 

"Invalid jQuery UI configuration. Selectable, draggable and droppable
components must be included."

Asikart QuickIcons 2.0.7
Joomla 3.8.2

Many thanks

**Comment by kingfisher64** [comment]
I've found out that System > Control Panel error is caused by Asikart QuickIcons Module 2.0.5.

As soon as that's disabled QuickIcons works again 2.0.7.

Still getting "Invalid jQuery UI configuration. Selectable, draggable and droppable
components must be included." dialog message though.

**Comment by asika32764** [comment]
Should be fixed in 2.0.8

**Comment by kingfisher64** [comment]
Thank you asika32764. Tried 2.0.8 and the first time it's installed (tried on two separate sites) I get internal 500 error dialog? Then when trying again it worked - both times?

**Comment by kingfisher64** [comment]
Further usages reveals:

Enabling the module to go into cpanel area the site again breaks on System > Control Panel on 2.0.8. 

I'll gladly test any fix :)

**Comment by asika32764** [comment]
2.0.9 Fixed the installation error.
https://github.com/asikart/quickicons/releases/tag/2.0.9

**Comment by kingfisher64** [comment]
Thanks I'm getting the following output on install:

"SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9 SELECT `asset_id` FROM `#__categories` WHERE `id` = 9"

System > Control Panel still breaking unfortunately (:

**Comment by asika32764** [comment]
The output is not a serious issue, it's a little bug from library, will be remove when library update in the future.

The System > Control Panel still works on my site, can you total remove module and it's folder then reinstall it?

**Comment by kingfisher64** [comment]
I've uninstalled and checked module and component folders and no previous asikart installations.

The problem is not the main component Simon for me :)

Whenever "Asikart QuickIcons Module" is enabled it breaks System > Control Panel display. As soon as it's disabled it works again.

**Comment by asika32764** [comment]
Then I guess it will be a extensions conflict, I cannot tell you what is the reason because I don't know what extensions you installed.

Can you try install Quickicons in a pure new Joomla site to check it?

**Comment by kingfisher64** [comment]
Just done this now Simon.

You are correct. www.kreativelizard.com/products/kl-admin-skin "Quick Icon - KL Admin Skin" conflicts with your plugin.

I can send you a copy of this (minus the licence key) for you to test?

**Comment by kingfisher64** [comment]
I will post an issue on their forum as well as it would be good if both plugins played well with each other given they both add very nice custom admin facilities.

But in essence, it's all working now. Only thing to fix is the install error message :)

Thanks for your work Simon.

"SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11 SELECT `asset_id` FROM `#__categories` WHERE `id` = 11"

**Comment by asika32764** [comment]
Yes, but I'm afraid that I have no time to fix for 3rd party extensions. I have some big project now.

The KL Admin is a paid extension, I think you can contact them to get their support and fix.

There was a similar case in 2014 because Akeeba uses an old API then break my extension.
https://github.com/asikart/quickicons/issues/5#issuecomment-51447878

**Comment by asika32764** [comment]
About the error message, it caused by a library which Quickicons used, it will be fixed soon after the library updated. Not Quickicons issue.

**Comment by kingfisher64** [comment]
Okay I understand. Thanks for the fixes :) 

I will contact KL. All it does it place a text link in sidebar so it's not needed really, could just be done with your plugin now everything is working.

I will post an update on here once it's fixed their end.

Kind regards

**Comment by kingfisher64** [comment]
Also if you update library please tell me and I will test for you to confirm :)

Thanks again.

**Comment by asika32764** [comment]
OK, I will keep this issue open after all issues fiexd.

**Comment by kingfisher64** [comment]
Just noticed another problem, unfortunately.

Asikart QuickIcons: Categories > Core Icons shows
`_Warning: JUser: :_load: Unable to load user with ID: 908_`

Also Backup Is Upto Date Icon is not displaying. All other icons are.

**Comment by asika32764** [comment]
This should be fixed if you install the latest version in a new site.

If this message still exists on current site, just change the created user at icon edit page.

**Comment by wbuk** [comment]
Thank you so much for fixing that.  It is really appreciated.  I wonder if you I can ask you to add a feature, which I did ask for over a year ago and you kindly said you would do it on your next update.  We have a lot of back-end users that use this as they need the joomla interface to be as simple as possible.  Your package is the best I have ever seen.  I really need a tool-tip, so when you hover your mouse over the button it brings up a tip in a box which we can customise.  This is really important as it gives additional information to the user without cluttering up the button.  Please could you add this feature. I would be most grateful as would many users.  Thank you.

**Comment by kingfisher64** [comment]
Tested on brand new install unfortunately still getting

 JUser: :_load: Unable to load user with ID: 908 

when clicking Categories > Core Icons

Please let me know if I can do anything to get this fixed. 

Kind regards

**Comment by asika32764** [comment]
@kingfisher64 Change the creator for every icons can remove this message.

@wbuk Actually I still have no time to create a new feature for quickcions, I'm only able to fix some problem if Joomla updated. New features will added after next major or minor version not patch  versions.
